app/controllers/EmailController.scala

```diff
@@ -120,6 +120,52 @@ class EmailController @Inject() (
       }
     }
 
+  val enterEmail: Action[AnyContent] = {
+      def displayEnterEmailDoYouWantToUse(journey: Journey.AfterAgreedTermsAndConditions)(implicit request: Request[_]): Result = {
```

Comment:
Tiny thing, but should this be `displayEnterEmailYouWantToUse`?

Reply:
whoops - yeah, that's not right. I changed it ]here](https://github.com/hmrc/essttp-frontend/pull/271/commits/92dafb20de96a6e57dd1496ad07bb78d5c0db613) now to `displayEnterEmailPage`